conf/messages.de

```diff
@@ -0,0 +1,19 @@
+# api
+
+api.basics.header = API-Grundlagen
+api.basics.search = Suche über alle Felder
+api.basics.field.simple = Einfache Feldsuche: "{0}"
+api.basics.field.nested = Geschachtelte Feldsuche: "{0}"
+api.basics.pagination = Suche mit Paginierung
+
+api.content_types.header = Inhaltstypen
+api.content_types.default = Standardmäßig liefert dieser Dienst strukturierte API-Antworten (als JSON):
+api.content_types.negotiate = Er unterstützt Content-Negotiation über den Accept-Header für JSON (application/json) oder HTML (text/html):
+api.content_types.override = Der Query-Parameter "format" kann verwendet werden, um den Accept-Header aufzuheben, z.B. zur Anzeige von JSON im Browser:
+
+api.jsonld.header = JSON-LD
+api.jsonld.info = JSON-API-Anfragen liefern JSON-LD (JSON for Linking Data). Um damit zu experimentieren können sie das JSON-LD oder URLs im <a href="{0}">JSON-LD Playground</a> einfügen.
```

Comment:
But here: https://en.wikipedia.org/wiki/Jsonld they reference https://www.w3.org/TR/json-ld-api/ which also name them "linked".
But it's not _so_ important for me.